core/src/engine/coro/pool.hpp

```diff
@@ -3,6 +3,8 @@
 #include <algorithm>  // for std::max
 #include <atomic>
 #include <cerrno>
+#include <iterator>
+#include <optional>
```

Comment:
Not needed anymore

Reply:
When we do try_deque we use std::optional, it's just that this include didn't exist before
https://github.com/userver-framework/userver/blob/develop/core/src/engine/coro/pool.hpp#L115